### ISSUE TITLE
R and Shiny Server Version

### ISSUE THREAD

**Issue by obenno** [swsoyee/2019-ncov-japan#5440]
Hi,

Thanks for the amazing website, may I ask what is the version of R and shiny server that your website was running in? Is the docker images available for public? Many thanks!

Oben

**Comment by swsoyee** [comment]
@obenno Sorry for the late response, I have notice the issue these days.

The R version is 4.0.2 (2020-06-22), shiny server is the open source one (https://rstudio.com/products/shiny/download-server/).
I'm not familiar with docker so I don't use docker in this project.

Best!

**Comment by obenno** [comment]
Thank you very much for the reply, that's very helpful!

Best,
Oben

**Comment by wzrzt** [comment]
@swsoyee  It looks great and works well. But I'm shocked that you are using shiny-server opensource without docker. I onced made a shiny app but found it was often down when there were more than 20 visiters.   
May I know how large is your data that the shiny directly loads, less than 10k rows of well calculated results data or some detail data which is calculated when shiny page is loading ?      
Have you saw what happens while more than 20 users visit it at the same time ?  
Hope for reply.   BTW,  I made a load balancer by nginx + several docker containers to make my shiny-server works for more visiters.
